.github/ISSUE_TEMPLATE/general-purpose.md

```diff
@@ -0,0 +1,47 @@
+Your issue may already be reported! Please search on the
+[issue tracker](https://github.com/elixir-cloud-aai/tus-storagehandler/issues) before creating
+one.
+
+## Expected Behavior
+
+<!--- If you're describing a bug, tell us what should happen -->
+
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+
+1.
+1.
+1.
+1.
```

Comment:
Remove these line, because there is nothing written in them, md formatter does this. IDK maybe its a bug of mdformatter.